package.json

```diff
@@ -10,7 +10,8 @@
     "test-wd": "mochify --wd test/client/**/*.spec.js",
     "test-cover": "mochify --cover test/client/**/*.spec.js",
     "test": "npm run phantom && npm run wd && npm run cover",
-    "test-server": "nodemon --watch test/server/**/*.js --watch sync-server.js --watch lib/sync-server.js test/server/test-server.js | bunyan"
+    "test-server": "nodemon --watch test/server/**/*.js --watch sync-server.js --watch lib/sync-server.js test/server/test-server.js | bunyan",
+    "eslint": "grunt"
```

Comment:
I think if we are adding a `eslint` npm command, it should be `grunt eslint` instead of the default.

You could add the `grunt eslint` command to the `test command`.